.github/workflows/coverage.yml

```diff
@@ -48,6 +48,7 @@ jobs:
         ./python -m venv .venv
         source ./.venv/bin/activate
         python -m pip install -U coverage
+        python -m pip install -r Misc/requirements-test.txt
```

Comment:
Another [imported comment](https://github.com/python/cpython/pull/20034#discussion_r424021474) from @zooba:

> Also as discussed, coverage is the one exception, but it's also not a core part of the PR workflow. We already switch CI providers for PRs every time Victor gets annoyed at network instability, so let's try and keep PR clean. We can leave it in the post-merge workflow if you really want.

My response is the same as to the first one. I'm OK with finding another workflow for getting the wheel, but if it's not simple I'd rather not bog this feature down with it.